bindings/apple/debug_build_library.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -eEu
+
+cd "$(dirname "$0")"
+
+echo "Running debug library build"
+
+# Path to the repo root
+SRC_ROOT=../..
```

Comment:
This is a totally optional (and possibly opinionated) nag but I've personally found it helpful to use lowercase for variable names in shell scripts – except for when I wanted to intentionally overwrite an environment variable. [This](https://stackoverflow.com/a/673940) SO answer makes a point about not lowercasing leading to accidental overrides but apart from that I also found it nice to be able to tell whether I _could_ override a variable's value from the environment just based on its casing in a script.

Reply:
Have switched to an xtask

Reply:
Even better! 👍